src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -3036,6 +3036,55 @@ module Types = struct
                     ~typ:(non_null int)
                 ]
       end
+
+      module ZkappCommandsDetails = struct
+        type input =
+          { account_creator : Signature_lib.Private_key.t
+          ; fee_payers : Signature_lib.Private_key.t list
+          ; num_of_zkapp_accounts : int
+          ; generating_new_accounts : int
+          ; transactions_per_second : float
+          ; duration_in_minutes : int
+          }
+
+        let arg_typ =
+          obj "ZkappCommandsDetails"
+            ~doc:"Keys and other information for scheduling zkapp commands"
+            ~coerce:(fun account_creator fee_payers num_of_zkapp_accounts
+                         generating_new_accounts transactions_per_second
+                         duration_in_minutes ->
+              Result.return
+                { account_creator
+                ; fee_payers
+                ; num_of_zkapp_accounts
+                ; generating_new_accounts
+                ; transactions_per_second
+                ; duration_in_minutes
+                } )
+            ~split:(fun f (t : input) ->
+              f t.account_creator t.fee_payers t.num_of_zkapp_accounts
+                t.generating_new_accounts t.transactions_per_second
+                t.duration_in_minutes )
+            ~fields:
+              Arg.
+                [ arg "accountCreator"
```

Comment:
Let's add maximum amount used in account creation and in transaction to be able to control the spending rate